src/Converter/Feature2Mesh.js

```diff
@@ -209,18 +200,24 @@ function featureToPoint(feature, options) {
                 normal.fromArray(feature.normals, v).multiply(inverseScale);
             }
 
-            coord.copy(context.setLocalCoordinatesFromArray(feature.vertices, v));
+            const localCoord = context.setLocalCoordinatesFromArray(feature.vertices, v);
             style.setContext(context);
             const { base_altitude, color, radius } = style.point;
-            coord.z = 0;
 
-            if (!pointMaterialSize.includes(radius)) {
-                pointMaterialSize.push(radius);
-            }
+            coord.copy(localCoord)
+                .applyMatrix4(context.collection.matrixWorld)
+                .as('EPSG:4326', coord);
 
             // populate vertices
-            base.copy(normal).multiplyScalar(base_altitude).add(coord).toArray(vertices, v);
+            base.copy(normal)
```

Comment:
shouldn't it be coord.z - base_altitude? (same for line and polygons)

Also a comment on what is computed here and about what is `base` would ease code reading

Reply:
1) I think there is a ambiguity of the meaning of base here. base is not the initial altitude but the altitude we want the base of the geometry to be set.'
Coord being the original altitude (before the altitude calculation), the shift we want to apply is indeed 'base_altitude - coord.z'.
Maybe we can rename base_altitude fot a less ambiguous name as ground_altitude (even if it's not really the ground)

2) I change the comment, hopefully it's more clear.

Reply:
ok thansk for the clarification